packages/capabilities/package.json

```diff
@@ -31,6 +31,22 @@
       "types": "./dist/src/*.d.ts",
       "import": "./src/*.js"
     },
+    "./filecoin": {
+      "types": "./dist/src/filecoin/index.d.ts",
+      "import": "./src/filecoin/index.js"
+    },
```

Comment:
nit: If the caps names have been decided with `filecoin/...` on the others I think this would make sense. But in current state, I think that it would be more consistent to expose `filecoin/storefront` here.

Reply:
Ok how about this: I've moved all the storefront caps into `storefront.js` and now `index.js` just re-exports the caps in `storefront.js` without the "filecoin" prefix. i.e. `filecoinOffer` => `offer`.

So then most people can do:

```js
import * as Filecoin from '@web3-storage/capabilities/filecoin'

Filecoin.offer
```

Which works with all our existing caps.

And equivalently:

```js
import * as Storefront from '@web3-storage/capabilities/filecoin/storefront'

Storefront.filecoinOffer
```

Reply:
cool, this sounds great!